Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -487,7 +487,12 @@ extension BrowserViewController: BrowserToolbarDelegate {
     func browserToolbarDidPressShare(browserToolbar: BrowserToolbarProtocol, button: UIButton) {
         if let selected = tabManager.selectedTab {
             if let url = selected.displayURL {
-                var activityViewController = UIActivityViewController(activityItems: [selected.title ?? url.absoluteString!, url], applicationActivities: nil)
+                let printInfo = UIPrintInfo(dictionary: nil)
+                printInfo.jobName = url.absoluteString
+                printInfo.outputType = .General
```

Comment:
@wesj I don't have AirPrint compatible printer either. Actually, according to the documentation, printInfo is unnecessary. UIKit assumes certain defaults. I have tried remove it locally and tracked the Printer Simulator logs. Only difference is made on jobName. I think JobName as a AbsoluteURL seems better If AirPrint compatible printer shows jobName.
![test](https://cloud.githubusercontent.com/assets/4068390/8278322/2518cc46-18d3-11e5-8059-d72c2cd5db83.png)